api/controllers/healthcheck.js

```diff
@@ -0,0 +1,19 @@
+const { createPool } = require('mariadb');
+const config = require('../config');
+const lib = require('../lib');
+
+exports.db = async (req, res) => {
+  let conn;
+  try {
+    const pool = createPool({ ...config.mysql, connectTimeout: 3000 });
+    conn = await pool.getConnection();
+    const q = `SELECT 1 as ok FROM Verse WHERE ID=1 LIMIT 1`;
+    const rows = await conn.query(q, []);
+    rows[0].ok = rows[0].ok === 1;
+    res.json(rows[0]);
+  } catch (err) {
```

Comment:
I'd prefer if we could wrap the different lines in try catch blocks. Have one big one swallows all the errors making it hard to determine what caused the error: the connection, pool, the query. 

For this end point it's fine to leave it as is but something to keep in mind.

Reply:
We should do this as a refactor of tech debt, I am working with what Navdeep Singh handed me. It would be difficult for me as a Novice JS dev to rewrite it that thoroughly.